docs/terminal/sydney.md

```diff
@@ -387,10 +392,27 @@ Heads-up coordination is not required from a SY TCU position to SRI for aircraft
 - Assigned 500ft above the BCTA as the CFL; and
 - Handed off to SRI
 
+!!! example
+    *FD214 is intending on leaving Class C airspace on descent into Bankstown.  The lower limit of CTA is `A045`.*  
+    **SDS:** "FD214 descend to 5,000ft, contact Sydney Radar 124.55"  
+    **FD214:** "Descend 5,000ft, 124.55, FD214"  
+
+    **FD214:** "Sydney Radar, FD214, descending 5,000ft"  
+    **SRI:** "FD214, Sydney Radar, cleared to leave controlled area descending, no reported IFR traffic"  
```

Comment:
Phraseology is "Leave controlled airspace descending". and just add area QNH too

Reply:
Copy.  I left out AQNH since they're already assigned A050 so assumed they were given it previously.

Reply:
hmm yeah fair point, i figured it would be more likely that theyve departed from an AD in the TMA like YSSY or YSBK where they got the local QNH, but could be either really, so yeah, fair to just omit it